src/Tile/Tile.stories.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import * as React from "react";
+import { storiesOf, setAddon } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import styles from "@sambego/storybook-styles";
```

Comment:
Is it really necessary for proper functioning ?

Reply:
Tom, do you mean `styles`? It's used in entire`storybook` to add some paddings :)

Reply:
OK, I checked change in `package.json` it's only ordering, I thought that we have new package.